src/main/webapp/script.js

```diff
@@ -53,7 +53,9 @@ function passQuery() {
 // Eg: Header for the query that finds for populations of all counties is
 // ["NAME","S0201_001E","state","county"]
 function displayVisualization(censusDataArray) {
-  if (censusDataArray[0][2] == 'state' && censusDataArray[0][3] != 'county') {
+  console.log(censusDataArray[0]);
+  const dataLength = censusDataArray[0].length;
```

Comment:
You don't seem to use this variable anywhere that I can see

Reply:
fixed!